doc/usecases/map_to_pdf.md

```diff
@@ -0,0 +1,27 @@
+# Use Case: Map to PDF
+
+## Summary
+
+- **Scope:** Map View
+- **Level:** User Goal
+- **Actors:** App User
+- **Brief:** Users can generate a PDF file of the map.
+- **Status:** Assigned
+- **Assignee:** Ramzan
+
+## Scenarios
+
+- **Precondition:**
+ - The user has opened the app and navigated to the desired map view.
+- **Main success scenario:**
+ - The app captures a screenshot of the current map view and generates a PDF file.
+ - The app adds a header/footer to the PDF containing the current time, map timestamp, GPS coordinates, and scale.
+ - The user receives the generated PDF file for printing or saving.
```

Comment:
How will this accomplished? 
I think the most reasonable thing would be an overview page with download links for each generated file.

Reply:
I would prefer if we can save it to Nextcloud. Then we could also easily implement features like automatic deletion https://apps.nextcloud.com/apps/files_retention

Btw. thanks for the reviews!